salt/_modules/metalk8s_service_configurator.py

```diff
@@ -0,0 +1,80 @@
+'''Metalk8s cluster and service configuration utility.'''
+
+import logging
+import re
+
+from salt.exceptions import CommandExecutionError
+
+import yaml
+from yaml.dumper import SafeDumper
+from yaml.representer import SafeRepresenter
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = 'metalk8s_service_configurator'
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def get_service_conf(
+    namespace,
+    configmap_name,
+):
+    """Reads  a ConfigMap from the name specified
+
+    Arguments:
+        configmap_name: Configmap name
+        namespace: the namespace where the configmap is found
+
+    Returns:
+        A dict of a specific service configuration
+
+    CLI Examples:
+
+    .. code-block:: bash
+
+        salt-call metalk8s_service_configurator.service_conf "metalk8s-monitoring" "metalk8s-prometheus-config"
+    """
+
+    if configmap_name is None:
+        raise Exception(
+            'Expected a configmap name but got {}'.format(configmap_name)
```

Comment:
```suggestion
            'Expected a ConfigMap name but got {}'.format(configmap_name)
```
Similar changes below. Be consistent.